server/Dockerfile

```diff
@@ -13,6 +13,16 @@ USER jboss
 
 RUN cd /opt/jboss/ && curl -L https://downloads.jboss.org/keycloak/$KEYCLOAK_VERSION/keycloak-$KEYCLOAK_VERSION.tar.gz | tar zx && mv /opt/jboss/keycloak-$KEYCLOAK_VERSION /opt/jboss/keycloak
 
+USER root
+
+#Give correct permissions when used in an OpenShift environment.
+RUN mkdir -p /opt/jboss/keycloak/standalone/log
+RUN mkdir -p /opt/jboss/keycloak/standalone/data
+RUN chown -R jboss:0 /opt/jboss/keycloak/standalone
+RUN chmod -R g+rw /opt/jboss/keycloak/standalone
```

Comment:
@stianst It does look a little weird, but I took the approach of doing the least number changes necessary to make it work.  These are the directories OpenShift was barking about, though making it $JBOSS_HOME and seeing what happens is an option...